web-ui/package-lock.json

```diff
@@ -0,0 +1,16581 @@
+{
```

Comment:
I think this file may need to be removed

Reply:
Should it be added to the .gitignore?

Reply:
Just deleted actually, we haven't had a package-lock thus far since we've been using yarn and I think Michael mentioned using yarn and npm like this could cause issues. Is that right @mkimberlin ?

Reply:
yep...should just be using yarn and this won't ever get generated